app/controllers/application_controller.rb

```diff
@@ -1,11 +1,21 @@
 class ApplicationController < ActionController::Base
-  before_action :set_current_user
+  # before_action :set_current_user
 
-  attr_reader :current_user
+  # attr_reader :current_user
 
-  private
+  # private
 
-  def set_current_user
-    @current_user = User.first
+  # def set_current_user
+  #   @current_user = User.first
+  # end
```

Comment:
- [ ] Well, in multiple files, I noticed that you have commented out code. This is not a good practice as commented-out code can be confusing and clutter the source files. If those lines of code are no longer necessary, please consider removing them completely to keep your codebase clean and maintainable. Thank you for your attention to code quality! 👍🧹📄